bundles/atom

```diff
@@ -1,8 +1,8 @@
 # [TITLE]: atom
-# [DESCRIPTION]: The popular atom text editor
+# [DESCRIPTION]: Text and source code editor with plug-in support for Node.js and Git. 
 # [STATUS]: Active
 # [CAPABILITIES]:
-# [TAGS]:
+# [TAGS]: Developer Tools, Productivity, editor, documentation
```

Comment:
Not sure I'd add the documentation tag to all the editor bundles but if this is sort of the suggestion for editor of choice for documentation that's fine with me.